src/segyviewlib/layoutcombo.py

```diff
@@ -94,6 +97,9 @@ def __init__(self, parent=None):
 
         self.currentIndexChanged.connect(self._layout_changed)
 
+    def set_selected_layout(self, index):
```

Comment:
It would be possible here to use `setCurrentIndex` directly - is there a reason not to? What's the argument here?